### PR TITLE
Merge dev brach to main branch

### PR DESCRIPTION
**Pull request by sankalpjain99**
Hi, can you please review my code?

### CODE REVIEW

**Review by Yash-VIT — APPROVED** [review]
Looks Good.

**Review by Yash-VIT — APPROVED** [review]
Looks Good.

**Review by Yash-VIT — APPROVED** [review]

**Comment by sankalpjain99** [comment]
Also added Eureka Service Discovery

File: backend/eureka-service/.gitignore [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+.settings
+.springBeans
+.sts4-cache
+
+### IntelliJ IDEA ###
+.idea
+*.iws
+*.iml
+*.ipr
+
```
Comment (Yash-VIT) [inline backend/eureka-service/.gitignore:23]: Add node_modules

Comment (sankalpjain99) [reply backend/eureka-service/.gitignore:23]: Resolved

**Review by Yash-VIT — APPROVED** [review]
Looks Good

**Pull request merged**